### ISSUE TITLE
Issue to install gitkube on EKS

### ISSUE THREAD

**Issue by fabiodouek** [hasura/gitkube#90]
When I run gitkube install, I'm getting the following error:

FATA[0000] installing gitkube failed: error creating CustomResourceDefinition: customresourcedefinitions.apiextensions.k8s.io is forbidden: User "system:anonymous" cannot create customresourcedefinitions.apiextensions.k8s.io at the cluster scope

**Comment by tirumaraiselvan** [comment]
Yes, this is a known issue: https://github.com/hasura/gitkube/issues/85 

There are 2 workarounds:
1) Create a new kubeconfig which is a proxy of the original one.
```
cat > ekskubeconfig.proxy <<EOF
apiVersion: v1
clusters:
- cluster:
    insecure-skip-tls-verify: true
    server: http://localhost:8080
  name: local-proxy 
- context:
    cluster: local-proxy
    user: ""
  name: local-proxy
kind: Config
EOF
```

And, then start the proxy:
```
kubectl proxy --port=8080 --kubeconfig=${PWD}/ekskubeconfig
```
Keep this proxy running and in a new shell use this kubeconfig by default.

```
export KUBECONFIG=${PWD}/ekskubeconfig.proxy
```

Now, `gitkube` cli should work.

2) The second way is to not use `gitkube` cli but directly use `kubectl` but this is little more complicated.

**Comment by fabiodouek** [comment]
Thanks @tirumaraiselvan , I tried that approach, but I getting the following error when I run: gitkube install

FATA[0000] installing gitkube failed: error creating CustomResourceDefinition: an error on the server ("") has prevented the request from succeeding (post customresourcedefinitions.apiextensions.k8s.io)

**Comment by tirumaraiselvan** [comment]
I am sorry. I have been very busy. I will fix this issue properly through this : https://github.com/hasura/gitkube/issues/85

**Comment by tirumaraiselvan** [comment]
This i now RELEASED in v3.0.0 . Just reinstall gitkube cli again.

`curl https://raw.githubusercontent.com/hasura/gitkube/master/gimme.sh | sudo bash`
